index.html

```diff
@@ -1,36 +1,37 @@
 <!DOCTYPE html>
 <html lang="en">
-  <head>
-    <meta charset="utf-8" />
 
-    <title>Calorie App</title>
-    <meta name="description" content="Calories application" />
-    <meta name="author" content="Bartlomiej Zdybel" />
-    <meta
-      name="viewport"
-      content="width=device-width, initial-scale=1, maximum-scale=1"
-    />
-    <link rel="stylesheet" type="text/css" href="./index.css" />
-    <script type="text/javascript" src="index.js"></script>
+<head>
+  <meta charset="utf-8" />
 
-    <link rel="icon" href="icon_code.jpg" />
-  </head>
+  <title>Calorie App</title>
+  <meta name="description" content="Calories application" />
+  <meta name="author" content="Bartlomiej Zdybel" />
+  <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />
+  <link rel="stylesheet" type="text/css" href="./index.css" />
+  <script type="text/javascript" src="index.js"></script>
 
-  <body>
-    <div class="container">
-      <div class="products-section">
-        <div class="products-section__title">Products</div>
-      </div>
+  <link rel="icon" href="icon_code.jpg" />
+</head>
+
+<body>
+  <div class="container">
```

Comment:
Bardziej semantycznie będzie jak zamienisz kolejno `divy`:
`.container` => `main`
`.section-box` => `section`
`.section-box__title` => `h*`

Reply:
css odrzuca `h*` (czcionka staje się czerwona a style nieaktywne)

Reply:
Jasne, jasne, chodziło mi o to, że zamiast `*` możesz wrzucić cokolowiek z przedziału 1-6 :P

Reply:
A, XD logiczne